src/components/shared/Settings.js

```diff
@@ -38,21 +30,22 @@ module.exports = React.createClass({
   },
 
   render: function() {
-    var editor = (store.get('editor')) ? store.get('editor') : 'ideditor';
+    const { editor, showSettings, closeSettings } = this.props;
+    console.log("SETTINGS", showSettings);
+    if (!showSettings) return null;
 
     return (
-      /* jshint ignore:start */
-      <div id='modal' className='animate modal modal-content active' onClick={this.onCancel}>
+      <div id='modal' className='animate modal modal-content active' onClick={closeSettings}>
```

Comment:
Fixed in [8cd1792](https://github.com/osmlab/to-fix/pull/160/commits/8cd1792727d20269801c7950b6821ec03c98f8cb).